pkg/events/parse_args.go

```diff
@@ -314,17 +323,232 @@ func (arg CustomFunctionArgument) Value() uint64 {
 	return arg.val
 }
 
-func parseBpfAttachHelperUsage(helperArg int32) (string, error) {
-	switch helperArg {
-	case 0:
-		return "false", nil
-	case 1:
-		return "true", nil
-	case 2:
-		return "unknown", nil
-	default:
-		return "", errfmt.Errorf("unknown value got from bpf_attach event for helper usage arg")
+func parseBpfAttachHelperUsage(helperList []uint64) ([]string, error) {
+
+	var usedHelpers []string
+
+	for idx, helperName := range BpfHelpers {
+		if (helperList[idx/64] & (1 << (idx % 64))) > 0 {
+			usedHelpers = append(usedHelpers, helperName)
+		}
 	}
+
+	return usedHelpers, nil
+}
+
+var BpfHelpers = []string{
```

Comment:
I still think we should have a GetAll() method from libbpfgo so you can range all helpers in `parseBpfAttachHelperUsage()`. I'm merging like it is now based on what Yaniv said, but I believe we should remove this slice from here (we should maintain it in libbpfgo because this list will likely suffer 1:1 updates with libbpf).